features/data_gov_uk.feature

```diff
@@ -44,3 +44,10 @@ Feature: Data.gov.uk
     And I force a varnish cache miss
     When I search for "" in datasets
     Then I should see an accurate dataset count
+
+  @high
+  Scenario: Check that we don't get any s3 CSP errors for organograms previews
+    Given I am testing "https://data.gov.uk"
+    When I preview an organogram
+    Then I can see the 'View the full organogram' button
```

Comment:
Here you can use `Then I should see "View the full organogram"` and you don't need the custom `Then` entry for this.